server/CGameHandler.cpp

```diff
@@ -1539,13 +1543,17 @@ void CGameHandler::save(const std::string & filename)
 
 	try
 	{
+		auto fileName = *CResourceHandler::get("local")->getResourceName(savePath);
 		{
-			CSaveFile save(*CResourceHandler::get("local")->getResourceName(savePath));
+			CSaveFile save(fileName);
 			saveCommonState(save);
 			logGlobal->info("Saving server state");
 			save << *this;
 		}
 		logGlobal->info("Game has been successfully saved!");
+#ifdef VCMI_HTML5_BUILD
+		html5::fsUpdate(fileName.c_str());
+#endif
```

Comment:
Should be done inside `createResource` method

Reply:
I want to do this only for saves/settings. Is it possible to determine that from `createResource`?

Reply:
No, but `createResource` should not be used in other cases anyway.